mesh/pmesh.cpp

```diff
@@ -2359,6 +2567,37 @@ int ParMesh::GetFaceNbrRank(int fn) const
    }
 }
 
+void
+ParMesh::GetFaceNbrElementFaces(int i, Array<int> &fcs, Array<int> &cor) const
+{
+   int n, j;
+   int el_nbr = i - GetNE();
+   if (face_nbr_el_to_face)
+   {
+      face_nbr_el_to_face->GetRow(el_nbr, fcs);
+   }
+   else
+   {
+      mfem_error("ParMesh::GetFaceNbrElementFaces(...) : "
+                 "face_nbr_el_to_face not generated.");
+   }
+   if (el_nbr < face_nbr_el_ori.Size())
+   {
+      const int * row = face_nbr_el_ori.GetRow(el_nbr);
+      n = fcs.Size();
+      cor.SetSize(n);
+      for (j=0; j<n; j++)
+      {
+         cor[j] = row[j];
+      }
+   }
+   else
+   {
+      mfem_error("ParMesh::GetFaceNbrElementFaces(...) : "
```

Comment:
I think in new code we should use MFEM_ABORT which is shorter and automatically adds the function name and line number etc.

Reply:
Good call